subset/password/brute_test/tmp/sshReport.txt

```diff
@@ -0,0 +1,13 @@
+Brute Test 

```

Comment:
why is this in tmp?   It's not a temporary file so.., I think the directory is misnamed.